packages/manfred2word/config/test/jest.js

```diff
@@ -4,4 +4,8 @@ export default {
   restoreMocks: true,
   moduleDirectories: ['<rootDir>/src', 'node_modules'],
   preset: 'ts-jest',
+
+  moduleNameMapper: {
```

Comment:
move __mocks__ into config folder and change to <rootDir>/config/__mocks__/fileMock.js